core/trino-main/src/test/java/io/trino/block/TestDictionaryBlock.java

```diff
@@ -227,34 +228,23 @@ public void testBasicGetPositions()
         assertEquals(((DictionaryBlock) dictionaryBlock).getDictionarySourceId(), dictionaryId);
 
         // out of range
+        final Block finalDictionaryBlock = dictionaryBlock;
         for (int position : ImmutableList.of(-1, 6)) {
```

Comment:
May be final Block finalDictionaryBlock can be declared in line 226?

Reply:
@MariyaR Could you explain more about the motive of declaring ```final Block finalDictionaryBlock``` in line 226? Thank you.

Reply:
Hi @choiwaiyiu, sorry, I mean in line 228, because the dictionaryBlock is assigned for the last time in this line and is not changed after that

Reply:
@MariyaR In my opinion, the code is more readable if we declare the variable close to where it is used.